test/unit/queue_tests.rb

```diff
@@ -68,7 +73,7 @@ class UnitTests < Assert::Context
       assert_equal expected, route.handler_class_name
     end
 
-    should "add jobs to redis using `add`" do
+    should "add jobs using `enqueue`" do
```

Comment:
@jcredding maybe use the actual `enqueue` method in this test?  Also maybe add assertions for using the alias?